eth/main.cpp

```diff
@@ -866,7 +866,7 @@ int main(int argc, char** argv)
 				cout << "  with seed as " << seedHash << endl;
 				if (valid)
 					cout << "(mixHash = " << r.mixHash << ")" << endl;
-				cout << "SHA3( light(seed) ) = " << sha3(EthashAux::light((uint64_t)bi.number)->data()) << endl;
+				cout << "SHA3( light(seed) ) = " << sha3(EthashAux::light(bi)->data()) << endl;
```

Comment:
Fair enough. But that means that this function should be deleted in general since we want to loosen the coupling between EthashAux and other modules. On it right now!